source/administrator/components/com_jedchecker/libraries/rules/jamss.php

```diff
@@ -86,11 +86,9 @@ protected function find($file)
 	 * In order to make this easy to update, no syntax changes are applied. Just store the variables in the object
 	 * for faster use.
 	 *
-	 * @param   unknown_type  $deepscan  Join the $jamssDeepSearchStrings
-	 *
-	 * @return  nothing
+	 * @return void
 	 */
-	private function init_jamss($deepscan = true)
+	private function init_jamss()
```

Comment:
Yes, I forgot to mention.
This is a unused parameter, and could be removed without consequences.
However, looking closer, I can enable it and make it working as it was in the mind of the original author.
It deserve a separate pull request, so that I've reverted this change here.